st2client/tests/unit/test_auth.py

```diff
@@ -22,9 +21,10 @@
 import requests
 import argparse
 import logging
+from cStringIO import StringIO
```

Comment:
Minor style / import ordering thing - we tend to put stdlib imports at the top then followed by 3rd party package imports then followed by local components imports.